### ISSUE TITLE
Why are the experimental results much higher than the results in the paper?

### ISSUE THREAD

**Issue by XXXxiaoxiaoxia** [qiaoguan/Person-reid-GAN-pytorch#24]
Why are the experimental results much higher than the results in the paper?

What is the difference between the loss function and the network structure in the code and the paper mentioned in the paper?
